src/ui/dialogs/spring_pane.rb

```diff
@@ -130,7 +130,20 @@ def update_dialog
     t = ERB.new(content)
 
     # display updated html
-    @dialog.set_html(t.result(binding))
+
+    # Normally, one would just use:
+    # @dialog.set_html(t.result(binding))
+    # but unfortunately, this gives the dialog focus, preventing the use of
+    # arrow keys to rotate the added object afterwards. (Try to comment
+    # this out, and then add a simple tetra, and try to scale it using arrow up)
+    # That's why we use javascript to replace the window content, which does not
+    # has that problem.
+    # TODO: When the added primitive contains a spring, rotation is still not
+    # possible
+
+    html = t.result(binding).dump
+    reload_command = "document.open();document.write(#{html});document.close();"
+    @dialog.execute_script reload_command
```

Comment:
@birneamstiel Maybe you could check if thats a) also happening for you on mac b) working fix for you c) if my comment is understandable 🗡️

Reply:
works on Mac, but see my notes below...

Reply:
comment is readable, but: is the `TODO` up to date? When I place a spring tetra on the ground I can rotate it...

Reply:
I am actually a little bit puzzled right now, because for me the rotation also works fine now. Maybe was a Sketchup restart issue, or I had some other stuff changed. I removed the comment now, and changed the hack +1